domain/model/pokemon.go

```diff
@@ -0,0 +1,44 @@
+package model
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	fileName string = "pokemons.csv"
+)
+
+type Pokemon struct {
```

Comment:
As a best practice, remember to add a description for every exported struct, interface, function, etc.
```go
// Pokemon contains .....
type Pokemon struct {
```
Please do this for all the exported stuff

Reply:
Agreed, comments added to all of the exported objects